### PR TITLE
Update local celery script

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

~Update README with local celery info~
Locally, have the celery worker also run beat. Update the documentation accordingly.

# Test instructions | Instructions pour tester la modification

- run the api
- run all celery with`make run-celery-local`
- see the beat logs coming through
- verify that sending still works

### CODE REVIEW

**Review by andrewleith — APPROVED** [review]
LGTM 🎉

File: README.md [hunk 1]
```diff
@@ -109,13 +109,9 @@ file. Copy that file to `.env` and customize it to your needs.
...
 ##  To run the queues
-```
-scripts/run_celery_local.sh
-```
+`make run-celery-local`
 
-```
-scripts/run_celery_beat.sh
-```
+`make run-celery-beat-local`
```
Comment (jimleroyer) [inline README.md:114]: It's great to mention the make target instead. As these are one more level of abstraction, we could even just have a one target named `make run-celeries-local`, as it doesn't have to be a one to one mapping with the actual scripts but rather intended usage (run celery setup locally).

**Review by jimleroyer — APPROVED** [review]
LGTM, merci!

I left a comment on potential improvement (on top of the improvement in this PR) but it's outside the scope of this change.

File: README.md (hunk 1 above)
Comment (sastels) [reply README.md:114]: hrm, running both the beat and celery workers with one make command? I didn't think that would work, but admittedly didn't try! I shall see!

Comment (sastels) [reply README.md]: oh wow, we can just add "--beat" to the worker script. That was easy.

**Comment by sastels** [comment]
Clearing approvals since I changed the scripts.

**Review by jimleroyer — APPROVED** [review]

**Pull request merged**